src/utils/deploy/upload-files.js

```diff
@@ -79,10 +85,12 @@ const retryUpload = (uploadFn, maxRetry) =>
 
     const tryUpload = async () => {
```

Comment:
The first argument is a counter, see https://github.com/MathieuTurcotte/node-backoff#event-ready could we use that? Only thing is that I think by reading the docs it starts at 0 for the first backoff, so we would need to do +1.

```suggestion
    const tryUpload = async (retryCount = 0) => {
```

Reply:
Hmm maybe I'm missing something, but I'm not seeing any parameters being passed to the function.

Reply:
https://github.com/MathieuTurcotte/node-backoff/blob/master/lib/backoff.js#L53 The event is passed some params, and as we use this function for the event we could use them.

Reply:
Right, but I'm logging the actual parameters that are passed to `tryUpload` and nothing comes up.

Reply:
Ah, my bad. I was looking in the wrong place. Will update!

Reply:
Done in https://github.com/netlify/cli/pull/4841/commits/6fed5241e61fa5f47903e7f25d1cca1ea1fa4e4b.